pkg/retriever/bitswapretriever.go

```diff
@@ -300,6 +315,7 @@ func (br *bitswapRetrieval) RetrieveFromAsyncCandidates(ayncCandidates types.Inb
 		TotalPayment:      big.Zero(),
 		NumPayments:       0,
 		AskPrice:          big.Zero(),
+		TimeToFirstByte:   time.Duration(ttfb.Load()),
```

Comment:
I should preserve this change even if this PR is closed, it's the only retriever to not return this value and there's no good reason not to